assets/src/legacy/action.js

```diff
@@ -255,91 +379,45 @@ var lizAction = function () {
+        if (actionButton.value == actionObjectUniqueId) {
+            actionButton.classList.add('btn-primary');
         }
 
         // Trigger the action when clicking on button
-        actionButton.click(function () {
-            // Clear the previous message
-            $('#lizmap-action-message').remove();
-
-            // Empty the action layer
-            let layer = lizMap.layers['actionLayer'];
-            layer.destroyFeatures();
-
-            // Get the layerId, featureId and action for this button
-            let val = $(this).val();
-            let vals = val.split('.');
-            let layerId = vals[0];
-            let featureId = vals[1];
-            let actionName = vals[2];
-
-            // Do nothing if the geometry was already set
-            // This allows to delete the current geometry
-            if (actionObjectUniqueId) {
-                // deactivate if the current action was this one
-                if (actionObjectUniqueId == layerId + '.' + featureId + '.' + actionName) {
-                    actionObjectUniqueId = null;
-                    return true;
-                }
-            }
-
-            // The action was not active, we can run it
-            actionObjectUniqueId = null;
-
-            // Get the action item data
-            let popupAction = getActionItemByName(actionName, Scopes.Feature, layerId);
-
-            // Display a confirm question if needed
-            if ('confirm' in popupAction && popupAction.confirm.trim() != '') {
-                let msg = popupAction.confirm.trim();
-                let go_on = confirm(msg);
-                if (!go_on) {
-                    return false;
-                }
-            }
-
-            // Toggle the given geometry
-            runLizmapAction(actionName, Scopes.Feature, layerId, featureId);
-
-            return false;
-        });
+        actionButton.addEventListener('click', popupActionButtonClickHandler);
 
         // Add hover interaction
-        actionButton.hover(
-            function () { $(this).addClass('btn-primary'); },
-            function () {
-                // Only remove btn-primary if the action is not active
-                // for this popup feature
-                if ($(this).val() != actionObjectUniqueId) {
-                    $(this).removeClass('btn-primary');
-                }
+        actionButton.addEventListener("mouseover", function (event) {
+            if (!event.target.matches('.popup-action')) return;
+            event.target.classList.add('btn-primary');
+        });
+        actionButton.addEventListener("mouseout", function (event) {
+            if (!event.target.matches('.popup-action')) return;
+            // Only remove btn-primary if the action is not active
+            // for this popup feature
+            if (event.target.value != actionObjectUniqueId) {
+                event.target.classList.remove('btn-primary');
             }
-        );
+        });
```

Comment:
This CSS part could help?
https://github.com/3liz/lizmap-web-client/blob/master/lizmap/www/assets/css/map.css#L2837